README.md

```diff
@@ -153,49 +176,45 @@ Each files has:
 - mime_type
 - original_file_url
 
-
-```php
-$files_raw = $api->request('GET', '/files/');
-```
-
-Previous request is just some raw request and it will return raw data from json.
-
-There's a better way to handle all the files by using method below.
-
-It will return an array of \Uploadcare\File objects to work with.
-
-This objects provide ways to display the file and to use methods such as resize, crop, etc
+The request we just made was a raw kind returning raw data.
+There's a better way to handle files, a method that returns an
+array of `\Uploadcare\File` to work with.
+`\File` objects provide better ways of displaying images and applying
+methods like resize, crop, etc.
 
 ```php
 $files = $api->getFileList();
 ```
 
-getFileList called without any params will return just an array of first 20 files objects (first page).
-
-But you can supply a page you want to see:
+`getFileList` called with no params returns an array of first 20
```

Comment:
`getFileList` called with no params returns an array of all records by 100 records in the chunk.